packages/round-manager/src/features/api/round.ts

```diff
@@ -271,9 +273,15 @@ export async function listRounds(
         roundMetadata,
         applicationMetadata,
         applicationsStartTime: new Date(round.applicationsStartTime * 1000),
-        applicationsEndTime: new Date(round.applicationsEndTime * 1000),
+        applicationsEndTime:
+          round.applicationsEndTime == ethers.constants.MaxUint256.toString()
+            ? maxDate
+            : new Date(round.applicationsEndTime * 1000),
```

Comment:
Does the name `applicationsXYZTime` come from code generation or is it something we have control over and introduced in this PR? If the latter, we should hint at the unit in the name itself, e.g. `applicationsEndTimeS`

Reply:
no, these dates were always like so. The only changes we've done was to set the proper values depending on the type of the round.